dockers/docker-platform-monitor/docker-pmon.supervisord.conf.j2

```diff
@@ -38,6 +38,19 @@ startsecs=0
 dependent_startup=true
 dependent_startup_wait_for=rsyslogd:running
 
+{% if not skip_chassisd %}
+[program:chassisd]
+command=/usr/local/bin/chassisd
+priority=3
+autostart=false
+autorestart=false
+stdout_logfile=syslog
+stderr_logfile=syslog
+startsecs=0
+dependent_startup=true
+dependent_startup_wait_for=start:exited
```

Comment:
I am trying to see if there is a scenario where the order in which we start the platform daemons matter ? like chassis daemon first and then followed by the other daemons like thermalctld. 
There could be data from DB populated by chassisd used by other pmon daemons like may be the slot number ?

Reply:
Good point. By design, we have kept the dependency loosely coupled. The get_my_slot() API is vendor specific and is bound to the chassis class rather than chassisd. For example, if thermalctld, wants to query some module state, it could directly call the chassis.module.get_status().
Since modules can go up/down, dependent processes have to cover for more dynamic scenarios. If there is a need in future, we can add the dependency during init.